### PR TITLE
chore: update localization strings via Crowdin

### PR DESCRIPTION
**Pull request by AndroidBob**
This PR pulls in the latest localization translations from Crowdin.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [833](https://10.10.124.134/job/AR-build-pipeline/833/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7610706)(7610706)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7610706)(7610706)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [844](https://10.10.124.134/job/AR-build-pipeline/844/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7615937)(7615937)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7615937)(7615937)-staging-compat.apk)

**Review by ohassine — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [849](https://10.10.124.134/job/AR-build-pipeline/849/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7616961)(7616961)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7616961)(7616961)-staging-compat.apk)

**Review by MohamadJaara — APPROVED** [review]

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1143,13 +1145,13 @@ registriert. Bitte versuchen Sie es mit einer anderen.</string>
     <string name="record_audio_max_file_size_reached_title">Aufnahme wurde beendet</string>
     <string name="record_audio_max_file_size_reached_text">Die Dateigröße für Audionachrichten ist auf %1$d MB begrenzt.</string>
     <string name="record_audio_unable_due_to_ongoing_call">Sie können während eines Anrufs keine Audionachricht aufnehmen.</string>
-    <string name="permission_dialog_title">App-Berechtigungen</string>
-    <string name="call_permission_dialog_description">Um einen Anruf zu tätigen, erlauben Sie Wire den Zugriff auf Ihr Mikrofon und die Suche nach Geräten in der Nähe in Ihren Geräteeinstellungen.</string>
-    <string name="label_not_now">Nicht jetzt</string>
-    <string name="last_message_composite_with_missing_text">sent an interactive message</string>
+    <string name="permission_dialog_title">App permissions</string>
+    <string name="call_permission_dialog_description">To make a call, allow Wire to access your microphone and to find nearby devices in your device settings.</string>
+    <string name="label_not_now">Not Now</string>
```
Comment (gongracr) [inline app/src/main/res/values-de/strings.xml:1150]: This is messing up German translations. Does anyone know where these changes might be coming from @MohamadJaara @ohassine ?

**Review by gongracr — CHANGES_REQUESTED** [review]

**Comment by AndroidBob** [comment]
Build [865](https://10.10.124.134/job/AR-build-pipeline/865/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7623465)(7623465)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7623465)(7623465)-staging-compat.apk)

**Pull request merged**